std_msgs/Header   header
uint8 	 device_channel
uint16	 can_id
uint8 	 can_dlc
uint8[]  can_data
